ext/drreg/drreg.c

```diff
@@ -432,8 +421,13 @@ drreg_event_bb_insert_late(void *drcontext, void *tag, instrlist_t *bb, instr_t
             "%s @%d." PFX " aflags=0x%x use=%d: lazily restoring aflags\n", __FUNCTION__,
             pt->live_idx, get_where_app_pc(inst), aflags, pt->aflags.in_use);
         res = drreg_restore_aflags(drcontext, bb, inst, pt, false /*keep slot*/);
-        if (res != DRREG_SUCCESS)
-            drreg_report_error(res, "failed to restore flags before app read");
+        if (res != DRREG_SUCCESS) {
+            LOG(drcontext, DR_LOG_ALL, 3,
```

Comment:
I would make this a lower verbosity, as a serious error.  Ditto below.

Reply:
Done